apps/condo/domains/banking/utils/testSchema/assets/1CClientBankExchangeToKeystoneObjects.js

```diff
@@ -0,0 +1,188 @@
+const dayjs = require('dayjs')
+const customParseFormat = require('dayjs/plugin/customParseFormat')
+dayjs.extend(customParseFormat)
+
+const PARSED_TRANSACTIONS_TO_COMPARE = [{
+  number: '61298',
+  date: dayjs('15.04.2022', 'DD.MM.YYYY', true),
+  amount: 700,
+  isOutcome: true,
+  purpose: 'Оплата лицензионного вознаграждения за использование базовой лицензии ПК Сервис Точка по тариф.пакету "Начало" за период с 01.04.22 по 30.04.22. Согласно Правилам ДО. НДС не предусмотрен.',
+  contractorAccount: {
+    'name': 'Точка ПАО Банка "ФК Открытие"',
+    'tin': '7706092528',
+    'number': '60322810700500000191',
+    'bankName': 'ТОЧКА ПАО БАНКА "ФК ОТКРЫТИЕ"',
+    'routingNumber': '044525999',
```

Comment:
Is that a personal data? IMO that's not good idea to store it as open-source part of the application

Reply:
This is an open data